sql/bank_test.go

```diff
@@ -95,7 +95,7 @@ CREATE TABLE IF NOT EXISTS bank.accounts (
 			update := `
 UPDATE bank.accounts
   SET balance = CASE id WHEN $1 THEN balance-$3 WHEN $2 THEN balance+$3 END
-  WHERE id IN ($1, $2)
+  WHERE id IN ($1, $2) AND true IN (SELECT balance >= $3 FROM bank.accounts WHERE id = $1)
```

Comment:
The `true IN` construct is needed because we don't properly handle subqueries used in non-tuple contexts that return only a single row. I'm filing an issue about this bug.